src/bin/fullnode.rs

```diff
@@ -97,14 +97,19 @@ fn main() {
     let keypair = Arc::new(keypair);
     let pubkey = keypair.pubkey();
 
+    let mut leader_scheduler = LeaderScheduler::default();
+
+    // Remove this line to enable leader rotation
+    leader_scheduler.use_only_bootstrap_leader = true;
```

Comment:
"bootstrap_leader" is used here, "first leader" is used elsewhere.  Let's use the same words everywhere.  "bootstrap leader" seems better than "first leader".